libs/ui/src/lib/carousel/carousel.component.ts

```diff
@@ -30,6 +31,7 @@ import { CarouselItem } from './carousel-item.directive';
 })
 export class GfCarouselComponent implements AfterContentInit {
   @ContentChildren(CarouselItem) public items!: QueryList<CarouselItem>;
+  itemsSignal = contentChildren('carouselItem', { read: ElementRef });
```

Comment:
Do we still need both `items` and `itemsSignal`?

Reply:
Is Keyboard focus and navigation `onKeyDown()` necessary?

Reply:
We can ditch the keyboard support altogether to simplify this component.